DANS-KNAW/easy-ingest-dispatcher
# isDepositReadyForIngest, warning on nonfatal throwable ->

NO JIRA ISSUE.

### Description
If a non-fatal Throwable is encountered during the check, besides returning false, now also logging a warning

### How to test
You could create a non-readable deposit directory and start the dispatcher, which should subsequently get an exception when trying to check it, and log a warning.
